WalletWasabi/Tor/Http/IHttpClient.cs

```diff
@@ -1,16 +1,43 @@
 using System;
 using System.Net.Http;
-using System.Security.Authentication;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace WalletWasabi.Tor.Http
 {
 	/// <summary>
-	/// Interface defining HTTP client capable of sending HTTP requests.
+	/// Interface defining HTTP client capable of sending HTTP requests that are relative to some base URI.
```

Comment:
This comment should be update once https://github.com/zkSNACKs/WalletWasabi/pull/5079/files#r564482276 is decided.